lib/slimmer/processors/tag_mover.rb

```diff
@@ -1,11 +1,11 @@
 module Slimmer::Processors
   class TagMover
     def filter(src, dest)
-      move_tags(src, dest, "script", dest_node: "body", keys: %w[src inner_html])
       move_tags(src, dest, "link",   must_have: %w[href])
       move_tags(src, dest, "meta",   must_have: %w[name content], keys: %w[name content http-equiv], insertion_location: :top)
       move_tags(src, dest, "meta",   must_have: %w[property content], keys: %w[property content], insertion_location: :top)
       move_tags(src, dest, "base",   must_have: %w[href])
+      move_tags(src, dest, "script", keys: %w[src inner_html], scripts_for_head: %w[async defer])
```

Comment:
```suggestion
      move_tags(src, dest, "script", keys: %w[src inner_html], head_if_attributes: %w[async defer])
```

Sorry one final suggestion! I think this suggestion makes more sense because in theory the element could be anything, not necessarily a `script` tag.

Reply:
very good point, I've renamed the method and other variables to make it all more generic